tensorboard/components/BUILD

```diff
@@ -5,12 +5,40 @@ load("//tensorboard/defs:vulcanize.bzl", "tensorboard_html_binary")
 
 licenses(["notice"])  # Apache 2.0
 
+tf_web_library(
+    name = "polymer_lib",
+    srcs = [
+        "polymer_lib.html",
+    ],
+    path = "/",
+    deps = [
+        ":analytics",
+        ":security",
+        "//tensorboard/components/tf_backend",
```

Comment:
:security, tf_backend, tf_storage, tf_tensorboard:registry were only present in the old `ng_polymer.html`.  Now that the new `polymer_lib.html` doesn't HTML-import them, can they be dropped from these BUILD deps?